packages/ui/src/components/SignIn/SignInFactorOnePasswordCard.tsx

```diff
@@ -78,8 +78,8 @@ export const SignInFactorOnePasswordCard = (props: SignInFactorOnePasswordProps)
           case 'complete':
             return setActive({
               session: res.createdSessionId,
-              navigate: ({ session }) => {
-                void navigateOnSetActive({ session, redirectUrl: afterSignInUrl });
+              navigate: async ({ session }) => {
+                return navigateOnSetActive({ session, redirectUrl: afterSignInUrl });
```

Comment:
Nitpick: No need for both async and returning the promise from `navigateOnSetActive` here.

I'd go with either `async + await` or just returning the promise from `navigateOnSetActive`.

Reply:
Oh forgot to remove that, cleaning this up now!